### PR TITLE
iistomin(TDI-38991): Improve test coverage for Netsuite

### PR DESCRIPTION
**Pull request by iistomin-gl**
**What is the current behavior?** (You can also link to an open issue here)
- Test coverage of Netsuite components is low (21%)

**What is the new behavior?**
- Test coverage of Netsuite components is acceptable (>= 60%)

**Please check if the PR fulfills these requirements**

- [X] The commit message follows Talend standard
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been added / updated (for bug fixes / features) ?
- [ ] The code coverage on new code >75%
- [X] The new code does not introduce new technical issues (sonar / eslint)

**What kind of change does this PR introduce?**

- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [X] Refactoring (no functional changes, no api changes)
- [ ] Build / CI related changes
- [X] Test coverage

**Does this PR introduce a breaking change?**

- [ ] Yes
- [X] No

**Other information**:
JIRA issue: https://jira.talendforge.org/browse/TDI-38991

### CODE REVIEW

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/676/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/676/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/680/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/680/)

**Comment by build-talend-common** [comment]
Pull request test result: [success](https://ci-common.datapwn.com/job/components_pull-request_master/712/) [:star:](https://ci-common.datapwn.com/job/components_pull-request_master/712/)

File: components/components-netsuite/components-netsuite-runtime/src/test/java/org/talend/components/netsuite/NetSuiteWebServiceMockTestFixture.java [hunk 1]
```diff
@@ -82,28 +82,48 @@ public NetSuiteWebServiceMockTestFixture(
...
+                        throw e;
+                    }
                 }
             }
-        }
 
-        URL wsdlLocation = new URL(portMockAdapter.getEndpointAddress().toString().concat("?wsdl"));
-//        assertNotNull(wsdlLocation.getContent());
+            URL wsdlLocation = new URL(portMockAdapter.getEndpointAddress().toString().concat("?wsdl"));
+            //        assertNotNull(wsdlLocation.getContent());
```
Comment (chmyga) [inline components/components-netsuite/components-netsuite-runtime/src/test/java/org/talend/components/netsuite/NetSuiteWebServiceMockTestFixture.java]: Commented code. Can be removed, I guess.

File: components/components-netsuite/components-netsuite-runtime/src/test/java/org/talend/components/netsuite/ValueConverterTest.java [hunk 2]
```diff
@@ -94,18 +96,22 @@ public void testEnumConverter() throws Exception {
         FieldDesc fieldDesc = typeDesc.getField("acctType");
         AvroConverter<Enum<AccountType>, String> converter1 =
                 (AvroConverter<Enum<AccountType>, String>) transducer.getValueConverter(fieldDesc);
-        assertEquals(AccountType.ACCOUNTS_PAYABLE.value(),
+        assertEquals(AvroUtils._string(), converter1.getSchema());
+        assertEquals(AccountType.class, converter1.getDatumClass());
+        Assert.assertEquals(AccountType.ACCOUNTS_PAYABLE.value(),
                 converter1.convertToAvro(AccountType.ACCOUNTS_PAYABLE));
-        assertEquals(AccountType.ACCOUNTS_PAYABLE,
+        Assert.assertEquals(AccountType.ACCOUNTS_PAYABLE,
```
Comment (chmyga) [inline components/components-netsuite/components-netsuite-runtime/src/test/java/org/talend/components/netsuite/ValueConverterTest.java]: Why have you added "Assert." here? it looks like assertEquals imported statically in this class.

**Pull request merged**